packages/core/aqua-to-js-compiler/src/utils.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { NonArrowType, ProductType } from '@fluencelabs/interfaces';
+import { readFile } from 'fs/promises';
+
+interface PackageJson {
+    version: string;
+    dependencies: {
+        ['@fluencelabs/aqua-api']: string
+    }
+}
+
+export async function getPackageJsonContent(): Promise<PackageJson> {
+    const content = await readFile('../package.json');
```

Comment:
never use slashes like this if you want your lib to be crossplatform

also reading file like this shouldn't work because relative path will be different dependening on where you run your lib from

why ditching json import?
If you don't wanna use json import - I would rather cadd a  build step to copy the content of package.json into a separate ts file that you will import normally

Reply:
Agree with point regarding slashes.

Reply:
This lib is node specific. You will run this lib after downloading npm package (or you have other ways to run it) ?
npm package provides you package.json in upper dir

Reply:
It's currently intended for Node. That is correct. But it won't work. See this https://nodejs.dev/en/api/v18/fs/#fspromisesreadfilepath-options

Reply:
Thanks, fixed.